src/allencell_ml_segmenter/_tests/prediction/test_file_input_widget.py

```diff
@@ -11,14 +12,18 @@
 from allencell_ml_segmenter.prediction.model import (
     PredictionModel,
 )
-from allencell_ml_segmenter.core.file_input_model import InputMode
+
+from allencell_ml_segmenter.core.file_input_model import (
+    InputMode,
+    FileInputModel,
+)
 
 MOCK_PATH: str = "/path/to/file"
 
 
 @pytest.fixture
-def prediction_model(qtbot: QtBot) -> PredictionModel:
-    return PredictionModel()
+def prediction_model(qtbot: QtBot) -> FileInputModel:
```

Comment:
This is confusing that the fixture called `prediction_model` returns a `FileInputModel`

Reply:
there's also more 'prediction' wording in this file that should prob be replaced since the widget isn't specific to prediction anymore